Source/DataModels/Program/Program.h

```diff
@@ -16,6 +17,7 @@ class Program
 
 	void BindUniformFloat4x4(const std::string& name, const float* data, bool transpose);
 	void BindUniformFloat3(const std::string& name, const float3 data);
+	void BindUniformFloat4(const std::string& name, const float4 data);
```

Comment:
```suggestion
	void BindUniformFloat4(const std::string& name, const float4& data);
```
Should do the same for the other methods, I guess it slipped by the original PR.